src/lib/apiTypesV2/SubscriptionExpression.h

```diff
@@ -40,8 +43,9 @@ struct SubscriptionExpression
   std::string               geometry;
   std::string               coords;
   std::string               georel;
+
+  StringFilter              stringFilter;
```

Comment:
Yes, this 'q' cannot be removed I'm afraid (I tried :-)).
A short comment to be added the the function header.